cspm/admin-guide/get-started-with-prisma-cloud/enable-access-prisma-cloud-console.adoc

```diff
@@ -14,13 +14,16 @@ So that Prisma Cloud can connect to your cloud environments for monitoring and y
 [#idcb6d3cd4-d1bf-450a-b0ec-41c23a4d4280]
 === NAT Gateway IP Addresses for Prisma Cloud
 Prisma™ Cloud uses the following NAT gateway IP addresses. To ensure that you can access Prisma Cloud and the API for any integrations that you enabled between Prisma Cloud and your incidence response workflows, or your agentless deployment or the Prisma Cloud Defenders to communicate with the Prisma Cloud Compute Console, review the list and update the IP addresses in your allow lists. 
+
 In the event of disruption due to a disaster, to help backup data in a timely manner, add the Disaster Recovery (DR) IP addresses to your allow lists.
 
+To add these IP addresses to an allow list, you may need to work with your network security team. The configuration for where you set up the allow list is dependent on your network architecture and it could be your firewall, proxy, or the server itself.
```

Comment:
I'm not sure this is necessary. I think most will know what it means to whitelist an IP.

Reply:
I was surprised too. I just was on a call for this. hence added it

Reply:
Give the users what they want then 😉